openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -252,6 +375,92 @@ private MediaType chooseResponseMediaType(ServerRequest req) {
         return resultMediaType;
     }
 
+    /**
+     * Extension we want SmallRye's OpenAPI implementation to use for parsing the JSON content in Extension annotations.
+     */
+    private static class HelidonAnnotationScannerExtension implements AnnotationScannerExtension {
+
+        @Override
+        public Object parseExtension(String key, String value) {
+
+            // Inspired by SmallRye's JsonUtil#parseValue method.
+            if (value == null) {
+                return null;
+            }
+
+            value = value.trim();
+
+            if ("true".equals(value) || "false".equals(value)) {
```

Comment:
Maybe `equalsIgnoreCase()`?

Reply:
I pretty much lifted this method from the corresponding SmallRye one, but I think your suggestion is a good one. Done.